Applications/SlicerApp/Testing/Python/DICOMReaders.py

```diff
@@ -97,23 +97,27 @@ def test_Part1DICOM(self):
         "fileName": "Mouse-MR-example-where-GDCM_fails.zip",
         "name": "Mouse-MR-example-where-GDCM_fails",
         "seriesUID": "1.3.6.1.4.1.9590.100.1.2.366426457713813178933224342280246227461",
-        "expectedFailures": ["GDCM", "Archetype"]
+        "expectedFailures": ["GDCM", "Archetype"],
+        "voxelValueQuantity": "(DCM, 110852, MR signal intensity)",
+        "voxelValueUnit": "(UCUM, [hnsf'U], Hounsfield unit)"
```

Comment:
Do you mean use units (as opposed to unit) everywhere or just here?
Do you know why in DICOM they use plural? In the standard it is stated that only one unit can be specified.

I'll fix the others.